### DISCUSSION TITLE
Deleting points bug

### DISCUSSION THREAD

**Discussion by dunc1ca** [Freika/dawarich#351] [General]
When deleting a point, a dialogue pops up saying "Are you sure you want to delete this point?" OK/Cancel.  When I click OK, the dialogue pops up again.  I have to click OK many times and eventually, the dialogue clears and the point is deleted.  I have tried this in both Chrome and Edge.

**Comment by Freika** [comment]
Huh, I thought I have fixed it... Will have another look 👍

**Reply by Freika** [reply]
Should be resolved in 0.19.7
